apps/hash-ai-worker-ts/src/activities/infer-entities/infer-entity-summaries.ts

```diff
@@ -76,12 +83,24 @@ export const inferEntitySummaries = async (params: {
       messages: completionPayload.messages,
     }),
     tools,
+    userAccountId,
+    graphApiClient,
+    linkUsageRecordToEntities: flowEntityId
+      ? [
+          {
+            linkEntityTypeId:
+              "https://hash.ai/@hash/types/entity-type/incurred-in/v/1",
+            entityId: flowEntityId,
+          },
+        ]
+      : [],
   });
 
   if (llmResponse.status !== "ok") {
     return {
       code: StatusCode.Internal,
       contents: [],
+      message: `Error getting LLM response: ${"message" in llmResponse ? llmResponse.message : llmResponse.status}`,
     };
   }
```

Comment:
@CiaranMn I think we should have a conversation at some point on how we plan to replace the `Status` return from the activities, so that we can more ergonomically return errors that can be returned by the `getLlmResponse` method. Ideally we could just directly return the possible errors, so there's no mapping between error objects required. I've created [this ticket](https://linear.app/hash/issue/H-2604/restructure-error-statuses-returned-by-flow-action-activities) for now